src/Drivers/nlpMDS_ex5.hpp

```diff
@@ -201,22 +201,22 @@ class Ex5 : public hiop::hiopInterfaceMDS
     return true;
   }
 
-  virtual bool eval_cons(const long long& n, const long long& m, 
-			 const long long& num_cons, const long long* idx_cons,  
+  virtual bool eval_cons(const long long& n, const long long& m,
+			 const long long& num_cons, const long long* idx_cons,
 			 const double* x, bool new_x, double* cons)
   {
     //return false so that HiOp will rely on the on-call constraint evaluator defined below
     return false;
   }
-  bool eval_cons(const long long& n, const long long& m, 
+  bool eval_cons(const long long& n, const long long& m,
 		 const double* x, bool new_x, double* cons)
   {
     const double* s = x+ns_;
     const double* y = x+2*ns_;
-    
```

Comment:
@nychiang : end of line is changed in very many places. This will make merging extremely difficult and laborious. I know why it happened ;) I suggest you somehow fix this.

Reply:
I tried to run command dos2unit, but it seems that I need to fix this issue manually. :-)

Reply:
ouch... maybe core.autocrlf settings of git will help?